queries/java/highlights.scm

```diff
@@ -258,6 +258,9 @@
 
 [ "(" ")" ] @punctuation.bracket
 
+(type_arguments . "<" @punctuation.bracket ">" @punctuation.bracket .)
```

Comment:
Is there any reason why it isn't `[ "<" ">" ] @punctuation.bracket` here? I'm pretty sure there isn't a necessity to anchor start and end node to this

Reply:
I was trying to protect it from any future syntax change, but that's probably overly careful and/or invalid. I'll use the simpler version you suggested :+1: